src/main/java/teammates/ui/controller/StudentFeedbackSubmissionEditPageAction.java

```diff
@@ -125,4 +125,9 @@ protected StudentAttributes getStudent() {
         
         return student;
     }
+    
+    protected boolean isRegisteredStudent(){
+        // a registered student must have an associated google Id
+        return (account.googleId != null) && (!account.googleId.isEmpty());
+    }
```

Comment:
Should this method be in the account object instead? Something like `isRegistered():boolean`?